From af51b5eed763718ccf512d5e21f6c079711a8759 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai.pt@gmail.com>
Date: Thu, 9 Jan 2025 14:38:01 +0800
Subject: [PATCH 1012/1016] ARM: dts: aspeed: catalina: Add Delta brick nodes

Add Delta brick nodes to support power sensors monitoring on PDB brick
board.

Signed-off-by: Potin Lai <potin.lai.pt@gmail.com>
---
 .../aspeed/aspeed-bmc-facebook-catalina.dts   | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
index dc41b08f60ec..9dba948282bd 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
@@ -581,6 +581,26 @@ vrm@62 {
 				compatible = "isil,raa228004";
 				reg = <0x62>;
 			};
+			delta_brick@63 {
+				compatible = "pmbus";
+				reg = <0x63>;
+			};
+			delta_brick@64 {
+				compatible = "pmbus";
+				reg = <0x64>;
+			};
+			delta_brick@65 {
+				compatible = "pmbus";
+				reg = <0x65>;
+			};
+			delta_brick@66 {
+				compatible = "pmbus";
+				reg = <0x66>;
+			};
+			delta_brick@67 {
+				compatible = "pmbus";
+				reg = <0x67>;
+			};
 		};
 		i2c1mux0ch4: i2c@4 {
 			#address-cells = <1>;
-- 
2.47.3

